Doc/whatsnew/3.9.rst

```diff
@@ -330,6 +339,13 @@ Optimizations
 Build and C API Changes
 =======================
 
+* Provide :c:func:`Py_EnterRecursiveCall` and :c:func:`Py_LeaveRecursiveCall`
+  as regular functions for the limited API. Previously, there were defined as
+  macros, but these macros didn't work with the limited API which cannot access
+  ``PyThreadState.recursion_depth`` field. Remove ``_Py_CheckRecursionLimit``
+  from the stable ABI.
+  (Contributed by Victor Stinner in :issue:`38644`.)
+
```

Comment:
Please drop these lines, they are added by accident.
Victor decided to move this text to another place.

Reply:
Thanks a lot! Fixed and also corrected typo in my own surname (what a shame).